fbsimctl/FBSimulatorControlKit/Sources/CommandParsers.swift

```diff
@@ -72,6 +72,36 @@ extension Parser {
   }
 }
 
+extension Configuration : Parsable {
+  public static func parser() -> Parser<Configuration> {
+    return Parser
+      .ofTwoSequenced(
+        self.optionsParser(),
+        self.controlConfigurationParser()
+      )
+      .fmap { (options, controlConfiguration) in
+        return Configuration(controlConfiguration: controlConfiguration, options: options)
+      }
+  }
+
+  public static func controlConfigurationParser() -> Parser<FBSimulatorControlConfiguration> {
+    return Parser.ofTwoSequenced(
```

Comment:
I'm pulling this styling from 'what looks ok', but I think reading downwards vertically with the indent level signifying the nesting of parsers, makes the most sense so i'll move `ofTwoSequenced` onto a new line